### ISSUE TITLE
ARMA Reforger

### ISSUE THREAD

**Issue by clementsX** [jetelain/Arma3Map#94]
Hello, are there any plans to add the REFORGER Maps to your site? 

 Currently you maintain the best available resource for Realism/Mil-sim, enabling us to properly place markers and plan operations. Adding this maps would be truly amazing.

**Comment by jetelain** [comment]
Hi,

It's a planned feature. I've built https://github.com/jetelain/GameMapStorage to be able to handle it. I still have to create the export process.

**Comment by OverlordZorn** [comment]
> Hi,
> 
> It's a planned feature. I've built https://github.com/jetelain/GameMapStorage to be able to handle it. I still have to create the export process.

Uhhh exciting!
Would this be also compatible with DCS Maps then?

**Comment by jetelain** [comment]
Yes I'm also willing to support DCS, but with lower resolution (as maps are very large)

**Comment by OverlordZorn** [comment]
> Yes I'm also willing to support DCS, but with lower resolution (as maps are very large)

that is very fair
